configs/example_video_config.yaml

```diff
@@ -0,0 +1,48 @@
+data:
+  videos_dir: /teamspace/studios/this_studio/lightning-action/data/videos  # MP4 directory
+  labels_dir: /teamspace/studios/this_studio/lightning-action/data/labels_numpy  # Numpy directory
+  expt_ids:
+  - 1.1.28R
+  - 1.1.28L
+  - 1.2.2R
+  - 1.2.28R
+  - 1.3.1R
+  - 1.4.1R
+  - 2.1.28L
+  - 3.1.28L
+  - 3.2.28L
+  ignore_index: -100
+  seed: 42
+model:
+  backbone: dtcn
+  dropout_rate: 0.1
+  input_size: 1536
```

Comment:
let's remove this and compute/extract from the encoder config. could be hard-coded for now, like if encoder is vitb then input_size is 1536, etc.
I want to look at using a resnet backbone soon and the input (latent) size will be defined in the config